db/schema.rb

```diff
@@ -0,0 +1,73 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20181022163932) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "games", force: :cascade do |t|
+    t.integer "game_id"
+    t.integer "turn_time"
+    t.datetime "start_time"
+    t.integer "turn_user_id"
+    t.integer "black_player_user_id"
+    t.integer "white_player_user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "moves", force: :cascade do |t|
+    t.integer "game_id"
+    t.integer "user_id"
+    t.integer "piece_id"
+    t.integer "vertical_end"
+    t.integer "horizontal_end"
+    t.integer "current_row"
+    t.integer "current_column"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "pieces", force: :cascade do |t|
+    t.integer "game_id"
+    t.integer "user_id"
+    t.string "piece_type"
```

Comment:
Hey! Can you change this to just be "type" instead of piece_type? my mentor said it will be better for me to write the inheritances